README.md

```diff
@@ -15,17 +18,24 @@ If you experience any issues with the older versions, you are most welcome to co
 
 ## Status
 
-Both the OPTIMADE specification and this repository are **under development**.
+Whilst v1.0 of the OPTIMADE specification has been released, this repository is **under development**.
+Outstanding features required for compliance with OPTIMADE v1.0 can be tracked with the OPTIMADE v1.0 label on [GitHub](https://github.com/Materials-Consortia/optimade-python-tools/issues?q=is%3Aopen+is%3Aissue+label%3A%22OPTIMADE+v1.0%22), which can be further filtered by backend.
+The package changelog can be found in [CHANGELOG.md](changelog.md)
 
-The latest stable version can be obtained from [PyPI](https://pypi.org/project/optimade) `pip install optimade` or by cloning the master branch of this repository `git clone git@github.com:Materials-Consortia/optimade-python-tools`.
+## Documentation
+
+This document, guides, and the full module API documentation can be found at [https://optimade.org/optimade-python-tools](https://optimade.org/optimade-python-tools).
 
 ## Installation
 
-Installation instructions, for both the index meta-database, and for the main API can be found in [the documentation](https://www.optimade.org/optimade-python-tools/install/).
+Installation instructions, for both the index meta-database, and for the main API can be found in [INSTALL.md](install.md).
+
+The latest stable version of this package can be obtained from [PyPI](https://pypi.org/project/optimade) `pip install optimade`.
+The latest development version of this package can be installed from the master branch of this repository `git clone git@github.com:Materials-Consortia/optimade-python-tools`.
 
 ## Contributing
 
-Contribution guidelines and tips can be found in [CONTRIBUTING.md](CONTRIBUTING.md).
+Contribution guidelines and tips can be found in [CONTRIBUTING.md](contributing.md).
```

Comment:
Same as above.
I see here that you converted the links from the actual filenames to this lowercase form. So perhaps it should stay that way? Huh.
```suggestion
Contribution guidelines and tips can be found in [the contribution documentation](CONTRIBUTING.md).
```

Reply:
Hmm, this was just for uniformity as the mkdocs links should be case-insensitive, but as you point out I think this breaks the GH rendered README, I'll sort it out